protocol/encryption/migrations/sqlite/1632236298_add_communities.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE communities_encryption (
```

Comment:
I'd not use the `communities` name anywhere in encryption, I would probably use `hash_ratchet` or similar